stack-citationapi/recipes/deploy-citation-data-api.rb

```diff
@@ -0,0 +1,27 @@
+node['deploy'].each do |application, deploy|
+
+  case application
+  when 'citation_apis'
+    next unless allow_deploy(application, 'citation_apis', 'citation-apis')
+  else
+    Chef::Log.info("deploy::internal-api - #{application} (in stack-citationapi) skipped")
```

Comment:
Log message should probably be `deploy-citation-data-api` or something, so it is easier to debug